test/Common.cpp

```diff
@@ -103,6 +103,8 @@ CommonOptions::CommonOptions(std::string _caption):
 		("no-smt", po::bool_switch(&disableSMT), "disable SMT checker")
 		("optimize", po::bool_switch(&optimize), "enables optimization")
 		("enforce-via-yul", po::bool_switch(&enforceViaYul), "Enforce compiling all tests via yul to see if additional tests can be activated.")
+		("enforce-gas-test", po::bool_switch(&enforceGasTest), "Enforce running gas tests on every test and adding expectations.")
+		("enforce-gas-test-min-value", po::value(&enforceGasTestMinValue), "Threshold value when enforcing gas tests.")
```

Comment:
Why not `--enforce-gas-cost`? That would match the variable name and we already have separate gas tests so it's not clear which gas tests this refers to.

```suggestion
		("enforce-gas-cost", po::bool_switch(&enforceGasTest), "Enforce gas cost expectations in semantic tests.")
		("enforce-gas-cost-min-value", po::value(&enforceGasTestMinValue), "Threshold value when enforcing gas cost expectations.")
```